tests/integration/test_data/npm_multiple_packages_lockfile3/bom.json

```diff
@@ -0,0 +1,1431 @@
+{
+  "bomFormat": "CycloneDX",
+  "components": [
+    {
+      "name": "accepts",
+      "properties": [
+        {
+          "name": "cachi2:found_by",
+          "value": "cachi2"
+        }
+      ],
+      "purl": "pkg:npm/accepts@1.3.8?vcs_url=git%2Bhttps://github.com/cachito-testing/npm-cachi2-multiple-packages.git%40c19e39bbd6588ee11ab3470a99763fcdbf49d1de#first_pkg/node_modules/accepts",
```

Comment:
And here as well :sweat_smile:

Reply:
Hmm, somehow all the "resolved" urls disappeared in https://github.com/cachito-testing/npm-cachi2-multiple-packages/commit/c19e39bbd6588ee11ab3470a99763fcdbf49d1de, that's probably why

Reply:
seems like regenerating files helped